drivers/scheduler/k8s/systemtestSpec/sysbench/px-sysbench-app.yml

```diff
@@ -0,0 +1,72 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: sysbench-mysql-data
+spec:
+  storageClassName: sysbench-sc-secure
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti 
+bash-3.2$ cat px-sysbench-app.yml 
```

Comment:
I guess this is not correct yaml

Reply:
Fixed this ...Because of copy and paste from shell